### ISSUE TITLE
Uninteresting Readme file 

### ISSUE THREAD

**Issue by SultanNoorudDin** [Ritchie88/Sign-Language-Translator#1]
your code works, great.

but a readme would be prefferred because there is still lots to understand in your code

A readme of CNN.py as an example :

---

# Sign Language MNIST Classification using Convolutional Neural Network (CNN)

This repository contains code for a Convolutional Neural Network (CNN) implemented in Python using Keras for the classification of Sign Language MNIST data.

## Overview

The main goal of this project is to demonstrate how to build, train, and evaluate a CNN model for classifying sign language gestures represented as images.

The dataset used for training and testing the model is the Sign Language MNIST dataset, which consists of grayscale images of hand gestures corresponding to letters of the alphabet (A-Z) and is structured similar to the MNIST dataset.

## Setup and Dependencies

To run the code, you'll need:

- Python (version 3.6 or later)
- Necessary libraries (NumPy, Pandas, Keras, scikit-learn)

Install the required libraries using pip:

```bash
pip install numpy pandas keras scikit-learn
```

## Usage

1. Download the Sign Language MNIST dataset.
2. Place the `sign_mnist_train.csv` and `sign_mnist_test.csv` files in the same directory as the code.
3. Run the `main()` function in the provided Python script (`main.py` or your chosen filename).

```bash
python main.py
```

## Code Structure

- `main.py`: Contains the main code for data preparation, model building, training, and evaluation.
- `README.md`: This file, providing information about the project and instructions for running the code.
- `SLmodel.h5`: The trained CNN model saved after training.

## Model Architecture

The CNN architecture consists of:

- Three sets of Conv2D layers with Batch Normalization and MaxPooling.
- Dropout layers for regularization.
- Fully connected Dense layers with ReLU activation and a final softmax layer for classification.

## Results

After training, the model's accuracy is evaluated on the test dataset, and the achieved accuracy is displayed in the console.

## Acknowledgments

The code structure and the CNN architecture used in this project are based on standard practices and common architectures for image classification tasks in deep learning.

---

Feel free to modify this README file as needed, adding more details or explanations specific to use case or any additional information relevant.
